test/test.ts

```diff
@@ -2101,6 +2101,17 @@ test('is.urlSearchParams', t => {
 	});
 });
 
+test('is.validDate', t => {
+	t.true(is.validDate(new Date()));
+	t.false(is.validDate(new Date('x')));
+	t.notThrows(() => {
+		assert.validDate(new Date());
+	});
+	t.throws(() => {
+		assert.validDate(new Date('x'));
+	});
```

Comment:
Needs a lot more assertions.

Reply:
Thanks for the feedback @sindresorhus!

Please help me understand what you are after. Do you feel like we needs more assertions to serve as examples/documentation? I mean, a date can only ever be in two states, either valid (represented by a number) or invalid (represented by `NaN`), so we can't really test it in more than two ways.